node-test/src/test/scala/org/bitcoins/node/BroadcastTransactionTest.scala

```diff
@@ -63,8 +63,18 @@ class BroadcastTransactionTest extends NodeUnitTest {
 
       bitcoindBalancePreBroadcast <- balanceF
       _ <- node.broadcastTransaction(tx)
-      _ <- TestAsyncUtil.awaitConditionF(() => hasSeenTx(tx),
-                                         duration = 1.second)
+      _ <- TestAsyncUtil
```

Comment:
So can this cause other test cases to fail if we can't reliably broadcast transactions to bitcoind?

Reply:
Yes, but on most other test cases we just broadcast the tx from the bitcoind node so we don't need to worry about that.